Tests/SwerverTests/RequestSpec/HttpRequestParserSpec.swift

```diff
@@ -11,10 +11,19 @@ class HttpRequestParserSpec: QuickSpec {
             it("returns an HttpRequest Object") {
                 let requestData = "GET / HTTP/1.1"
                 let parsedRequest = httpRequestParser.parse(request: requestData)
-                expect(parsedRequest.getHttpVersion()).to(equal(HttpVersion.current))
-                expect(parsedRequest.getUrl()).to(equal(URL(string: "/")))
-                expect(parsedRequest.getMethod()).to(equal(RequestMethod.get))
+                expect(parsedRequest?.getHttpVersion()).to(equal(HttpVersion.current))
```

Comment:
This should be split into multiple tests. One for each attribute

Reply:
will look into that

Reply:
fixed